src-tool/Pact/Analyze/Types/Capability.hs

```diff
@@ -19,19 +19,27 @@ import           Pact.Analyze.PactSFunArray (eitherArray, mkPactSFunArray)
 import           Pact.Analyze.Types.Shared
 import           Pact.Analyze.Types.Types
 
+
+data CapabilityAnnot
+  = CapManaged
+  | CapEvent
+  deriving Show
+
 -- | The "signature" for a capability in Pact -- a family of tokens.
 data Capability where
-  Capability :: SingList schema -> CapName -> Capability
+  Capability :: SingList schema -> CapName -> Maybe CapabilityAnnot -> Capability
 
 instance Show Capability where
-  showsPrec p (Capability sch capName) = showParen (p > 10) $
+  showsPrec p (Capability sch capName capAnnot) = showParen (p > 10) $
       showString "Capability "
     . showsPrec 11 sch
     . showChar ' '
     . showsPrec 11 capName
+    . showChar ' '
+    . showsPrec 11 capAnnot
```

Comment:
Does the show instance for analysis' `Capability` end up on chain? (In that case, would printouts of a `Capability` mentioned in a model be a forking change?

Reply:
everything in `src-tool/` is related to the FV module and does not have impact to on-chain stuff